### ISSUE TITLE
how to set default pageSizeMenu to different number

### ISSUE THREAD

**Issue by rassemdev** [TerryZ/v-page#2]
Now current default pageSizeMenu is 10. But i wanna change it to 3. How can i do it?

**Comment by TerryZ** [comment]
```html
<v-page :setting="pagePhoto"></v-page>
```

```js
        data(){
            return {
                pagePhoto: {
                    totalRow: arr.length,
                    pageSizeMenu:[3, 10, 20],
                    language: 'en'
                }
            };
        }
```

set `pageSizeMenu` option, default setting is `[10, 20, 50, 100]`, you can change this Array data, the first number in Array will be the pageSize of v-page

**Comment by rassemdev** [comment]
I did set it that way. but its only taking 10 as default. Not taking other numbers

**Comment by TerryZ** [comment]
@rassemdev emmmmm.. that was a bug, i have already fixed, please update to 1.5.0 version
